test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,7 +90,8 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        if(inputCommand.equals(AddCommand.COMMAND_WORD) || inputCommand.equals(ClearCommand.COMMAND_WORD) || inputCommand.equals(DeleteCommand.COMMAND_WORD))
+            assertEquals(addressBook, saveFile.load());
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation:exclamation: One or more of these problems noticed: 
1. extra/missing spaces
2. brace style problem
3. missing braces
4. name doesn't follow naming convention
5. line too long